includes/Data/Options.php

```diff
@@ -48,6 +48,7 @@ final class Options {
 		'page_on_front'                 => 'page_on_front',
 		'theme_settings'                => 'theme_settings',
 		'flow_preset'                   => 'flow_preset',
+		'wp_seo_social'                 => 'wp_seo_social',
```

Comment:
Shouldn't this be
```suggestion
		'wpseo_social'                 => 'wpseo_social',
```
?